docs/pages/guides/meshery-metrics.md

```diff
@@ -6,6 +6,7 @@ permalink: guides/meshery-metrics
 language: en
 type: Guides
 category: performance
+language: en
```

Comment:
This already has `language` no need to put it again.

Reply:
done